opennms-dao/src/main/java/org/opennms/netmgt/dao/support/ServiceResourceType.java

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2007-2015 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2015 The OpenNMS Group, Inc.
+ *
```

Comment:
use 2021 the two lines above or was this file just moved around?

Reply:
Changed.